specification/visualstudio/resource-manager/Microsoft.VisualStudio/2014-04-01-preview/examples/CreateProjectResource.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "parameters": {
+    "body": {
+        "id": "/subscriptions/0de7f055-dbea-498d-8e9e-da287eedca90/resourceGroups/VS-Example-Group/providers/Microsoft.VisualStudio/account/ExampleAccount/project/ExampleProject",
+        "name": "ExampleProject",
+        "type": "Microsoft.VisualStudio/account/project",
+        "location": "Central US",
+        "tags": {},
+        "properties": {
+          "ProcessTemplateId": "6B724908-EF14-45CF-84F8-768B5384DA45",
```

Comment:
Thanks, I'll give that a try.

Is it possible to use the `enum` semantics on non-string types? ProcessTemplateId is actually a GUID, and the only accepted values are 6B724908-EF14-45CF-84F8-768B5384DA45, ADCC42AB-9882-485E-A3ED-7678F01F66BC, 27450541-8E31-4150-9947-DC59F998FC01. 

From looking at other RP specs and the Swagger docs, I don't see a way to assign a friendly name to these (the IDs correspond to Scrum, Agile and Cmmi process templates).